### PR TITLE
app-admin/rasdaemon: git build based on upstream ebuilds

### PR DESCRIPTION
**Pull request by maxeaubrey**
Yes, `git` is insecure, but no https afaict.

### CODE REVIEW

**Comment by maxeaubrey** [comment]
Upstream conf.d file also had a typo, gotta keep everyone on their toes.

**Comment by maxeaubrey** [comment]
Fair point, should use last commit date instead of when i'm doing things. Amended.

**Comment by patricklauer** [comment]
Optimally you'd use a snapshot tarball instead of using git for fetching things ...

**Comment by maxeaubrey** [comment]
Sure, i'll fix it up tomorrow to be like that

**Comment by maxeaubrey** [comment]
@patricklauer done.

File: app-admin/rasdaemon/files/rasdaemon.confd [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# pass the --record option if sqlite is enabled and you wish store events in sqlite
+#RASDAEMON_ARGS=--record
```
Comment (wiktor-b) [inline app-admin/rasdaemon/files/rasdaemon.confd]: this fix should find its way upstream

File: app-admin/rasdaemon/rasdaemon-20190626.ebuild [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools linux-info systemd vcs-snapshot
```
Comment (wiktor-b) [inline app-admin/rasdaemon/rasdaemon-20190626.ebuild]: is there a need for autotools and vcs-snapshot?

File: app-admin/rasdaemon/rasdaemon-20190626.ebuild [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	)
+"
+
+pkg_setup() {
+	linux-info_pkg_setup
+	local CONFIG_CHECK="~ACPI_EXTLOG"
+	check_extra_config
+}
+
+src_prepare() {
```
Comment (wiktor-b) [inline app-admin/rasdaemon/rasdaemon-20190626.ebuild]: does the default src_prepare() not work?

Comment (maxeaubrey) [reply app-admin/rasdaemon/rasdaemon-20190626.ebuild]: It does not, hence the addition.

File: app-admin/rasdaemon/rasdaemon-20190626.ebuild (hunk 2 above)
Comment (maxeaubrey) [reply app-admin/rasdaemon/rasdaemon-20190626.ebuild]: vcs-snapshot sets the work path:
```
 * ERROR: app-admin/rasdaemon-20190626::adjust failed (prepare phase):
 *   The source directory '/var/tmp/portage/app-admin/rasdaemon-20190626/work/rasdaemon-20190626' doesn't exist
```

autotools is for reconf/conf:
```
>>> Preparing source in /var/tmp/portage/app-admin/rasdaemon-20190626/work/rasdaemon-20190626 ...
/var/tmp/portage/app-admin/rasdaemon-20190626/temp/environment: line 1697: eautoreconf: command not found
/var/tmp/portage/app-admin/rasdaemon-20190626/temp/environment: line 1698: eautoconf: command not found
```

File: app-admin/rasdaemon/files/rasdaemon.confd (hunk 1 above)
Comment (maxeaubrey) [reply app-admin/rasdaemon/files/rasdaemon.confd]: Strong agree.

File: app-admin/rasdaemon/rasdaemon-20190626.ebuild (hunk 3 above)
Comment (wiktor-b) [reply app-admin/rasdaemon/rasdaemon-20190626.ebuild]: weird, the ebuild in ::gentoo just has default src_prepare().

**Pull request merged**